pep-0680.rst

```diff
@@ -191,31 +191,36 @@ Basing on another TOML implementation
 
 Potential alternatives include:
 
-* ``tomlkit``.
+* ``tomlkit``
+
   ``tomlkit`` is well established, actively maintained and supports TOML v1. An
```

Comment:
This is better than how it was, but since all items but one begin with the project name and then repeat it as the first word of the first sentence, perhaps consider just doing:

```suggestion
* ``tomlkit`` is well established, actively maintained and supports TOML v1. An
```

(and the same for the others, perhaps with a tweak to the opening line if you think its important to be strictly correct, such as `Several potential alternative implementations exist:`